src/components/HomePage/HomePage.tsx

```diff
@@ -67,6 +69,7 @@ export default class HomePage extends React.PureComponent<Props, State> {
     const { isAnimationPlaying } = this.state
     const projects = Object.values(this.props.projects)
     const templates = getTemplates()
+    const shouldRenderPromo = ethereum && !ethereum.isDapper
```

Comment:
this should be `!ethereum || ! ethereum.isDapper`, otherwise this will only render the promo to MetaMask users, but not to non-web3 users.